chef/data_bags/crowbar/migrate/network/101_add_ironic_network.rb

```diff
@@ -0,0 +1,20 @@
+def upgrade(ta, td, a, d)
+  unless a["networks"].key? "ironic"
+    a["networks"]["ironic"] = ta["networks"]["ironic"]
+  end
+
+  # copy intf1 as intf3
+  a["conduit_map"].each do |cm|
+    unless cm["conduit_list"].key? "intf3"
+      cm["conduit_list"]["intf3"] = cm["conduit_list"]["intf1"]
```

Comment:
This was the best option I could think of... admin is on `intf0` and I want to avoid `intf3` being on the same interface, so making a copy of the second conduit will take the second NIC in case of `dual` mode and with one NIC it's not going to work anyway.

I'm not sure what will happen if there's no entry in the `conduit_map` matching the network some barclamp is trying to allocate an IP from.